pkg/engine/apicall/apiCall.go

```diff
@@ -82,6 +82,18 @@ func (a *apiCall) Execute(ctx context.Context, call *kyvernov1.APICall) ([]byte,
 }
 
 func (a *apiCall) transformAndStore(jsonData []byte) ([]byte, error) {
+	if jsonData == nil {
+		if a.entry.APICall.Default.Raw == nil {
+			return nil, fmt.Errorf("default is missing: failed to add resource data to context entry %s", a.entry.Name)
```

Comment:
If the apiCall returns a 200 but a `nil` response, should this be considered an error?

Reply:
no it should not be considered an error, it should be a valid value, fixed it